app/views/layouts/searches_redesign.html.haml

```diff
@@ -14,3 +15,5 @@
     = stylesheet_link_tag 'searches_redesign'
   %body
     #main-content= yield
+    = render "/shared/google_tag_manager_noscript"
+    = render "/shared/affiliate_google_analytics"
```

Comment:
Should we move this outside of the `<body>`?

Reply:
thanks @ddzz - historically we've had it in the body, but looking at https://github.com/digital-analytics-program/gov-wide-code/blob/master/documentation/GSA%20DAP%204.1%20-%20Quick%20Guide.pdf, they recommend it goes in the head. sorry @krbhavith !